src/todos/dto/create-todo.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { IsString, IsNotEmpty, IsOptional } from 'class-validator'
+import { Types } from 'mongoose'
+
+export class CreateTodoDto {
+  _id: Types.ObjectId
```

Comment:
If we allow passing `_id` during todo creation, we need to validate it. If not — no need to define this prop.

Reply:
Fixed